src/main/java/TrainingEvaluator.java

```diff
@@ -0,0 +1,50 @@
+public class TrainingEvaluator {
+
+    public int evaluateLengthOfTraining (int timeOfTrainingInMinutes) {
+        if (timeOfTrainingInMinutes < 30) {
```

Comment:
3 metody powtarzają taki sam if {...} else {...}. Zastanów się czy da się taki kod wyodrębnić do osobnej metody tak aby nie musieć go powtarzać.

Reply:
Zastosowałem takie rozwiązanie ale dla 2 metod. Trzecia metoda ma odwrócona punktacje, chyba ze można to zrobić jeszcze inaczej?